### PR TITLE
[READY TO MERGE] more flavor text fixes round 2

### PR DESCRIPTION
**Pull request by Wizardcrying**
revival of old PR
fixes some flavor text stuff and also triggers celt/fixes two bugs related to the KEEPER AI module's name and examine text (the module of which shouldn't be tangible but is on box centcomm anyway, why)

update: fixed the removal of flavor

### CODE REVIEW

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 1]
```diff
@@ -216,15 +216,15 @@
...
-			desc = "An all black space pod with no insignias."
+			desc = "A space pod painted black."
 		if("pod_mil")
-			desc = "A dark grey space pod brandishing the Nanotrasen Military insignia"
+			desc = "A dark grey space pod bearing the Nanotrasen military insignia."
 		if("pod_synd")
-			desc = "A menacing military space pod with Fuck NT stenciled onto the side"
+			desc = "A menacing military space pod with \"Fuck NT\" stenciled onto the side."
 		if("pod_gold")
-			desc = "A civilian space pod with a gold body, must have cost somebody a pretty penny"
```
Comment (ghost) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm:225]: >Removing fun

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 2]
```diff
@@ -357,17 +357,17 @@
 */
 	for(var/mob/living/carbon/slime/M in range(1,usr))
 		if(M.Victim == usr)
-			to_chat(usr, "You're too busy getting your life sucked out of you.")
+			to_chat(usr, "You attempt to enter \the [src.name], but [M.name] pulls you back!")
```
Comment (ghost) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: >Removing fun

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 3]
```diff
@@ -357,17 +357,17 @@
...
 
-	visible_message("<span class='notice'>[usr] starts to climb into [src.name]</span>")
+	visible_message("<span class='notice'>[usr] starts to climb into [src.name].</span>")
 
 	if(enter_after(40,usr))
 		if(!src.occupant)
 			moved_inside(usr)
 		else if(src.occupant!=usr)
-			to_chat(usr, "[src.occupant] was faster. Try better next time, loser.")
+			to_chat(usr, "[src.occupant] has already entered the pod.")
```
Comment (ghost) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: >Removing fun

File: code/game/objects/items/weapons/cigs_lighters.dm [hunk 4]
```diff
@@ -407,7 +407,7 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/cigar
 	name = "Premium Cigar"
-	desc = "A brown roll of tobacco and... well, you're not quite sure. This thing's huge!"
+	desc = "A large roll of tobacco and chemicals."
```
Comment (ghost) [inline code/game/objects/items/weapons/cigs_lighters.dm]: >Removing fun

File: code/modules/admin/admin.dm [hunk 5]
```diff
@@ -786,13 +786,13 @@ var/global/floorIsLava = 0
 			<BR>
 			<A href='?src=\ref[src];secretsfun=hardcore_mode'>[ticker&&ticker.hardcore_mode ? "Disable" : "Enable"] hardcore mode (makes starvation kill!)</A><BR>
 			<A href='?src=\ref[src];secretsfun=tripleAI'>Triple AI mode (needs to be used in the lobby)</A><BR>
-			<A href='?src=\ref[src];secretsfun=eagles'>Egalitarian Station Mode</A><BR>
+			<A href='?src=\ref[src];secretsfun=eagles'>Egalitarian Station Mode (all-access for everyone)</A><BR>
```
Comment (ghost) [inline code/modules/admin/admin.dm]: It removes door access, not gives all access

File: code/modules/clothing/shoes/magboots.dm [hunk 6]
```diff
@@ -102,7 +102,7 @@
 //Syndicate
 /obj/item/clothing/shoes/magboots/syndie
 	name = "blood-red magboots"
-	desc = "Reverse-engineered magnetic boots that have a heavy magnetic pull. Property of Gorlex Marauders."
```
Comment (Skullyton) [inline code/modules/clothing/shoes/magboots.dm]: >Removing [LORE]

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 7]
```diff
@@ -88,7 +88,7 @@
 
 /mob/living/simple_animal/crab/snowy
 	name = "Snowy"
-	desc = "While you'd think that most crabs in cold climates would stick to the relatively warmer water, this one's adapted to living on the land and has camouflage to boot"
+	desc = "A crab with a white shell."
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: >Removing fun

File: code/modules/reagents/reagent_dispenser.dm [hunk 8]
```diff
@@ -255,7 +255,7 @@
 
 /obj/structure/reagent_dispensers/beerkeg
 	name = "beer keg"
-	desc = "A beer keg"
+	desc = "A massive keg containing beer."
```
Comment (ghost) [inline code/modules/reagents/reagent_dispenser.dm]: May not necessarily contain beer

File: code/datums/uplink_item.dm [hunk 9]
```diff
@@ -362,7 +362,7 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/dangerous/revolver
 	name = "Fully Loaded Revolver"
-	desc = "A traditional handgun which fires .357 rounds. Has 7 chambers. Can down an unarmoured target with two shots."
+	desc = "A traditional handgun which fires .357 rounds. Has 7 chambers. Can down an unarmored target with two shots."
```
Comment (BestName) [inline code/datums/uplink_item.dm]: asd

File: code/game/gamemodes/cult/cult_items.dm [hunk 10]
```diff
@@ -103,10 +103,10 @@
 	siemens_coefficient = 0
 
 /obj/item/clothing/suit/space/cult
-	name = "cult armour"
+	name = "cult armor"
```
Comment (BestName) [inline code/game/gamemodes/cult/cult_items.dm]: asd

File: code/game/gamemodes/cult/cult_items.dm [hunk 11]
```diff
@@ -103,10 +103,10 @@
...
 	icon_state = "cult_armour"
 	item_state = "cult_armour"
-	desc = "A bulky suit of armour, bristling with spikes. It looks space proof."
+	desc = "A bulky suit of armor covered in spikes. It looks space proof."
```
Comment (BestName) [inline code/game/gamemodes/cult/cult_items.dm]: asd

File: code/game/gamemodes/cult/talisman.dm [hunk 12]
```diff
@@ -127,7 +127,7 @@
 <A href='?src=\ref[src];rune=runestun'>Fuu ma'jin</A> - Allows you to stun a person by attacking them with the talisman.<BR>
 <A href='?src=\ref[src];rune=soulstone'>Kal om neth</A> - Summons a soul stone<BR>
 <A href='?src=\ref[src];rune=construct'>Da A'ig Osk</A> - Summons a construct shell for use with captured souls. It is too large to carry on your person.<BR>"}
-//<A href='?src=\ref[src];rune=armor'>Sa tatha najin</A> - Allows you to summon armoured robes and an unholy blade<BR> //Kept for reference
+//<A href='?src=\ref[src];rune=armor'>Sa tatha najin</A> - Allows you to summon armored robes and an unholy blade<BR> //Kept for reference
```
Comment (BestName) [inline code/game/gamemodes/cult/talisman.dm:130]: asd

File: code/game/objects/structures/crates_lockers/closets/syndicate.dm [hunk 13]
```diff
@@ -137,7 +137,7 @@
 		return
 
 /obj/structure/closet/vox_raiders
-	name = "vox armoury closet"
+	name = "vox armory closet"
```
Comment (BestName) [inline code/game/objects/structures/crates_lockers/closets/syndicate.dm]: asd

File: code/game/objects/structures/crates_lockers/closets/syndicate.dm [hunk 14]
```diff
@@ -1,5 +1,5 @@
 /obj/structure/closet/syndicate
-	name = "armoury closet"
+	name = "armory closet"
```
Comment (BestName) [inline code/game/objects/structures/crates_lockers/closets/syndicate.dm]: asd

File: code/game/objects/structures/crates_lockers/closets/syndicate.dm [hunk 15]
```diff
@@ -154,7 +154,7 @@
 
 
 /obj/structure/closet/vox_raiders/trader
-	name = "vox armoury closet"
+	name = "vox armory closet"
```
Comment (BestName) [inline code/game/objects/structures/crates_lockers/closets/syndicate.dm]: asd

File: code/modules/clothing/spacesuits/alien.dm [hunk 16]
```diff
@@ -26,7 +26,7 @@
 	slowdown = 3
 
 
-// Vox space gear (vaccuum suit, low pressure armour)
+// Vox space gear (vaccuum suit, low pressure armor)
```
Comment (BestName) [inline code/modules/clothing/spacesuits/alien.dm]: asd

File: code/modules/clothing/spacesuits/alien.dm [hunk 17]
```diff
@@ -53,7 +53,7 @@
 	name = "alien pressure suit"
 	icon_state = "vox-pressure"
 	item_state = "vox-pressure"
-	desc = "A huge, armoured, pressurized suit, designed for distinctly nonhuman proportions."
+	desc = "A huge, armored, pressurized suit, designed for distinctly nonhuman proportions."
```
Comment (BestName) [inline code/modules/clothing/spacesuits/alien.dm]: asd

File: code/modules/clothing/spacesuits/alien.dm [hunk 18]
```diff
@@ -63,10 +63,10 @@
 	eyeprot = 3
 
 /obj/item/clothing/suit/space/vox/carapace
-	name = "alien carapace armour"
+	name = "alien carapace armor"
```
Comment (BestName) [inline code/modules/clothing/spacesuits/alien.dm]: asd

File: code/modules/clothing/spacesuits/alien.dm [hunk 19]
```diff
@@ -63,10 +63,10 @@
...
 	icon_state = "vox-carapace"
 	item_state = "vox-carapace"
-	desc = "An armoured, segmented carapace with glowing purple lights. It looks pretty run-down."
+	desc = "An armored, segmented carapace with glowing purple lights. It looks pretty run-down."
```
Comment (BestName) [inline code/modules/clothing/spacesuits/alien.dm]: asd

File: code/modules/clothing/spacesuits/alien.dm [hunk 20]
```diff
@@ -88,7 +88,7 @@
 	desc = "An alien helmet with enormous goggled lenses."
 
 /obj/item/clothing/suit/space/vox/medic
-	name = "alien armour"
+	name = "alien armor"
```
Comment (BestName) [inline code/modules/clothing/spacesuits/alien.dm]: asd

File: code/modules/clothing/spacesuits/alien.dm [hunk 21]
```diff
@@ -123,7 +123,7 @@
 
 /obj/item/clothing/shoes/magboots/vox
 
-	desc = "A pair of heavy, jagged armoured foot pieces, seemingly suitable for a velociraptor."
+	desc = "A pair of heavy, jagged armored foot pieces."
```
Comment (BestName) [inline code/modules/clothing/spacesuits/alien.dm]: asd

**Review by BestName — APPROVED** [review]
Patriotic/10
God bless America.
bald-eagle-crying

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 2 above)
Comment (Exxion) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm]: `\the [src.name]` doesn't work.

File: code/modules/admin/admin.dm (hunk 5 above)
Comment (Intigracy) [reply code/modules/admin/admin.dm]: All door access but command and sec iirc

**Comment by ghost** [comment]
>make a grammar PR

>misspell words 

Shaking my head to be honest

File: code/modules/clothing/shoes/magboots.dm (hunk 6 above)
Comment (Intigracy) [inline code/modules/clothing/shoes/magboots.dm:105]: At least describe it as being a tag on the tongue.

File: code/modules/reagents/reagent_dispenser.dm (hunk 8 above)
Comment (Intigracy) [reply code/modules/reagents/reagent_dispenser.dm]: labeled as or something

**Review by Intigracy — DISMISSED** [review]

File: code/game/gamemodes/cult/talisman.dm (hunk 12 above)
Comment (ghost) [reply code/game/gamemodes/cult/talisman.dm:130]: It's even commented out, the heck is this?

Comment (BestName) [reply code/game/gamemodes/cult/talisman.dm:130]: The good ol' find and replace.

File: code/modules/clothing/spacesuits/alien.dm (hunk 21 above)
Comment (Skullyton) [inline code/modules/clothing/spacesuits/alien.dm]: Damn it celt, stop trying to force the irish to be relevant

**Comment by Wizardcrying** [comment]
I wasn't lying when I wrote

>triggers celt

Now I can make that

>triggers celt AND bestname

File: code/game/gamemodes/cult/talisman.dm (hunk 12 above)
Comment (Wizardcrying) [reply code/game/gamemodes/cult/talisman.dm:130]: I did it just to piss Celt off. I didn't use find and replace.

File: code/modules/admin/admin.dm (hunk 5 above)
Comment (Wizardcrying) [reply code/modules/admin/admin.dm]: thanks

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 2 above)
Comment (Exxion) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: I was being serious. It should be `\the [src]` and `\the [M]`

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 22]
```diff
@@ -357,17 +357,17 @@
 */
 	for(var/mob/living/carbon/slime/M in range(1,usr))
 		if(M.Victim == usr)
-			to_chat(usr, "You're too busy getting your life sucked out of you.")
+			to_chat(usr, "You attempt to enter \the [src.name], but [M.name] pulls you back!")
 			return
 //	to_chat(usr, "You start climbing into [src.name]")
 
-	visible_message("<span class='notice'>[usr] starts to climb into [src.name]</span>")
+	visible_message("<span class='notice'>[usr] starts to climb into [src.name].</span>")
```
Comment (Exxion) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: `\the [src]`

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 23]
```diff
@@ -343,7 +343,7 @@
 		if(occupant == usr)
 			move_outside(usr)
 		else
-			to_chat(usr, "<span class='notice'><B>The [src.name] is already occupied!</B></span>")
+			to_chat(usr, "<span class='notice'><B>\The [src.name] is already occupied!</B></span>")
```
Comment (Exxion) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: Remove `.name`

**Review by Exxion — CHANGES_REQUESTED** [review]

**Comment by Wizardcrying** [comment]
god damn it @Skullyton you commented on the removal and not the addition

File: code/modules/clothing/shoes/magboots.dm (hunk 6 above)
Comment (Skullyton) [inline code/modules/clothing/shoes/magboots.dm]: Lore has been restored

File: code/game/gamemodes/cult/talisman.dm (hunk 12 above)
Comment (Wizardcrying) [reply code/game/gamemodes/cult/talisman.dm:130]: ok

**Review by ghost — CHANGES_REQUESTED** [review]
It's spelled armour tbh

**Comment by Wizardcrying** [comment]
@Exxion I fixed em

**Comment by 9600bauds** [comment]
why are you REMOVING flavor
god what the fuck is this whole PR

File: code/game/objects/items/devices/flash.dm [hunk 24]
```diff
@@ -1,6 +1,6 @@
 /obj/item/device/flash
 	name = "flash"
-	desc = "Used for blinding and being an asshole."
+	desc = "Used for blinding and incapacitation. Can be directed at an individual or used to disorient an area."
```
Comment (9600bauds) [inline code/game/objects/items/devices/flash.dm]: whyyy

it can't even disorient an area

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 25]
```diff
@@ -1,6 +1,6 @@
 /obj/structure/closet/secure_closet/medical1
 	name = "Medicine Closet"
-	desc = "Filled with medical junk."
+	desc = "A closet containing medical supplies."
```
Comment (9600bauds) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: whyy

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 26]
```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A space pod painted black."
```
Comment (9600bauds) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: this is a case of a description that might as well not be there

why are you replacing descriptions with something that might as well not be there

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 27]
```diff
@@ -357,17 +357,17 @@
 */
 	for(var/mob/living/carbon/slime/M in range(1,usr))
 		if(M.Victim == usr)
-			to_chat(usr, "You're too busy getting your life sucked out of you.")
+			to_chat(usr, "You're too busy getting the life sucked out of you by [M.name]!")
```
Comment (9600bauds) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: should this really be using .name, and shouldn't it be using \the?

File: code/game/gamemodes/cult/cult_items.dm [hunk 28]
```diff
@@ -103,10 +103,10 @@
 	siemens_coefficient = 0
 
 /obj/item/clothing/suit/space/cult
-	name = "cult armour"
+	name = "cult armor"
 	icon_state = "cult_armour"
 	item_state = "cult_armour"
-	desc = "A bulky suit of armour, bristling with spikes. It looks space proof."
```
Comment (9600bauds) [inline code/game/gamemodes/cult/cult_items.dm]: bristling is such a good adjective and I can't fathom why are you removing it

File: code/game/objects/items/devices/flash.dm (hunk 24 above)
Comment (Wizardcrying) [reply code/game/objects/items/devices/flash.dm]: I meant the in-hand interaction. What do you suggest as a replacement?

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm (hunk 25 above)
Comment (Wizardcrying) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: Explain.

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 26 above)
Comment (Wizardcrying) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm]: I don't know how having no insignias is worth inclusion. I'll make the text more descriptive, though.

File: code/game/gamemodes/cult/cult_items.dm (hunk 28 above)
Comment (Wizardcrying) [inline code/game/gamemodes/cult/cult_items.dm]: agreed, fixed

**Comment by 9600bauds** [comment]
God dammit and this is part 2?

**Comment by ghost** [comment]
Every single time I look at this PR The plus to minus ratio goes down and down and down

**Comment by Wizardcrying** [comment]
stop shitposting and tell me what's actually wrong
I agreed with you, @9600bauds, in some cases when you actually said what was wrong instead of just saying "why"
@DrCelt doesn't help either and nor does @BestName 
Bauds abusing collab status doesn't help either

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 1 above)
Comment (Exxion) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm:225]: I'd personally say keep the original wording and just fix the comma splice and missing period.

File: code/game/machinery/alarm.dm [hunk 29]
```diff
@@ -166,6 +166,7 @@
 /obj/machinery/alarm/proc/first_run()
 	area_uid = areaMaster.uid
 	name = "[areaMaster.name] Air Alarm"
+	desc = "An alarm used to control the area's atmospherics systems."
```
Comment (Exxion) [inline code/game/machinery/alarm.dm]: Put this up in the object definition rather than this proc. The name has to be here because it involves a variable, but this is  always the same.

File: code/game/objects/items/devices/flash.dm (hunk 24 above)
Comment (Exxion) [reply code/game/objects/items/devices/flash.dm]: I suggest you leave this iconic description how it was

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm (hunk 25 above)
Comment (Exxion) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: Maybe this one's iconic too? I don't play medbay

File: code/modules/admin/admin.dm [hunk 30]
```diff
@@ -786,13 +786,13 @@ var/global/floorIsLava = 0
 			<BR>
 			<A href='?src=\ref[src];secretsfun=hardcore_mode'>[ticker&&ticker.hardcore_mode ? "Disable" : "Enable"] hardcore mode (makes starvation kill!)</A><BR>
 			<A href='?src=\ref[src];secretsfun=tripleAI'>Triple AI mode (needs to be used in the lobby)</A><BR>
-			<A href='?src=\ref[src];secretsfun=eagles'>Egalitarian Station Mode</A><BR>
+			<A href='?src=\ref[src];secretsfun=eagles'>Egalitarian Station Mode (removed access on doors except for Command and Security)</A><BR>
```
Comment (Exxion) [inline code/modules/admin/admin.dm]: Change it to "removes" to keep consistency with other verb descriptions.

File: code/modules/admin/admin.dm [hunk 31]
```diff
@@ -804,24 +804,24 @@ var/global/floorIsLava = 0
 			<A href='?src=\ref[src];secretsfun=striketeam'>Send in a strike team</A><BR>
 			<A href='?src=\ref[src];secretsfun=placeturret'>Create a turret</A><BR>
 			<BR>
-			<A href='?src=\ref[src];secretsfun=traitor_all'>Everyone is the traitor</A><BR>
-			<A href='?src=\ref[src];secretsfun=onlyone'>There can be only one!</A><BR>
+			<A href='?src=\ref[src];secretsfun=traitor_all'>Make everyone traitors</A><BR>
+			<A href='?src=\ref[src];secretsfun=onlyone'>Highlander Mode (There can be only one!)</A><BR>
```
Comment (Exxion) [inline code/modules/admin/admin.dm]: Technically this button is also used for Wizard Wars, but whatever

File: code/modules/admin/admin.dm [hunk 32]
```diff
@@ -804,24 +804,24 @@ var/global/floorIsLava = 0
...
+			<A href='?src=\ref[src];secretsfun=floorlava'>The floor is lava! (WARNING: extremely lame and DANGEROUS!)</A><BR>
 			<BR>
-			<A href='?src=\ref[src];secretsfun=massbomber'>Turn every players into Bomberman</A><BR>
+			<A href='?src=\ref[src];secretsfun=massbomber'>Turn all players into Bomberman</A><BR>
 			<A href='?src=\ref[src];secretsfun=bomberhurt'>Make Bomberman Bombs actually hurt players</A><BR>
-			<A href='?src=\ref[src];secretsfun=bomberdestroy'>Make Bomberman Bombs actually destroy stuff</A><BR>
-			<A href='?src=\ref[src];secretsfun=bombernohurt'>Make Bomberman Bombs harmless to players(default)</A><BR>
+			<A href='?src=\ref[src];secretsfun=bomberdestroy'>Make Bomberman Bombs actually destroy structures</A><BR>
+			<A href='?src=\ref[src];secretsfun=bombernohurt'>Make Bomberman Bombs harmless to players (default)</A><BR>
 			<A href='?src=\ref[src];secretsfun=bombernodestroy'>Make Bomberman Bombs harmless to the environnement(default)</A><BR>
```
Comment (Exxion) [inline code/modules/admin/admin.dm]: While you're here, add a space to this one, too

File: code/modules/mob/living/simple_animal/hostile/faithless.dm [hunk 33]
```diff
@@ -1,6 +1,6 @@
 /mob/living/simple_animal/hostile/faithless
 	name = "The Faithless"
-	desc = "The Wish Granter's faith in humanity, incarnate"
+	desc = "A being of shadow. It posesses a menacing gaze and a tortured voice."
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/faithless.dm:3]: OK this one was probably iconic too, but to be fair yours is better in this case

**Review by Exxion — CHANGES_REQUESTED** [review]
Most of these are opinion, so feel free to ignore them if most people disagree

File: code/game/machinery/alarm.dm (hunk 29 above)
Comment (Wizardcrying) [reply code/game/machinery/alarm.dm]: Whoops, nice catch

File: code/modules/admin/admin.dm (hunk 32 above)
Comment (Wizardcrying) [reply code/modules/admin/admin.dm]: Didn't notice the `environnement` either. Thanks.

Comment (Exxion) [reply code/modules/admin/admin.dm]: I didn't either, somehow

**Comment by ghost** [comment]
>+2/-9

>Not controversial

**Comment by ghost** [comment]
>I did it just to piss Celt off

>Please give me feedback of why you don't like it Celt 

🤔

**Comment by Exxion** [comment]
He changed most of the things people were downvoting about
Probably
You can't really tell what someone's upset about if they only vote

**Comment by Wizardcrying** [comment]
@DrCelt you said that my change of colour to color in the last pr was non-atomic

**Review by Exxion — APPROVED** [review]

**Comment by 9600bauds** [comment]
hi yes i believe there is plenty of controversy here

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm (hunk 25 above)
Comment (9600bauds) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: This is just downgrading a snarky description (for something that needs no description) into a useless and bland description (for something that needs no description)

**Comment by 9600bauds** [comment]
This has actually improved since the last time I looked at it so I'm removing my :-1:

I realize now that my "whyy" comment was really shitty and uninformative. I apologize for that, and I left a little better explanation for my comments.

**Comment by Wizardcrying** [comment]
what fucking votes
fuck off with that

**Comment by PJB3005** [comment]
I'm against removing non-descriptive descriptions on objects that can go without it.

File: code/game/objects/items/weapons/shields.dm [hunk 34]
```diff
@@ -132,7 +132,7 @@
 
 /obj/item/weapon/cloaking_device
 	name = "cloaking device"
-	desc = "Use this to become invisible to the human eyesocket."
+	desc = "A device that completely hides a user."
```
Comment (PJB3005) [inline code/game/objects/items/weapons/shields.dm]: Eeeh.

File: code/modules/clothing/spacesuits/alien.dm (hunk 21 above)
Comment (PJB3005) [inline code/modules/clothing/spacesuits/alien.dm]: eeeh

File: code/game/objects/items/weapons/shields.dm (hunk 34 above)
Comment (Wizardcrying) [reply code/game/objects/items/weapons/shields.dm]: "Use this to become invisible to the human eye"?

File: code/modules/mob/living/simple_animal/hostile/faithless.dm (hunk 33 above)
Comment (PJB3005) [reply code/modules/mob/living/simple_animal/hostile/faithless.dm:3]: Yeah eeeh on this one.

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 35]
```diff
@@ -78,7 +78,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/mini_uzi
 	name = "Uzi"
-	desc = "A lightweight, fast firing gun, for when you want someone dead. Uses .45 rounds."
+	desc = "A lightweight, fast firing gun. Uses .45 rounds."
```
Comment (PJB3005) [inline code/modules/projectiles/guns/projectile/automatic.dm]: eeeeh

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 36]
```diff
@@ -95,7 +95,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/c20r
 	name = "\improper C-20r SMG"
-	desc = "A lightweight, fast firing gun, for when you REALLY need someone dead. Uses 12mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp"
+	desc = "A lightweight, fast firing gun. Uses 12mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
```
Comment (PJB3005) [inline code/modules/projectiles/guns/projectile/automatic.dm]: eeeeeh

**Comment by Shadowmech88** [comment]
>fixes some flavor text stuff

That's a funny way to say "unnecessarily makes a bunch of descriptions clinical and boring."

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 36 above)
Comment (PJB3005) [reply code/modules/projectiles/guns/projectile/automatic.dm]: Also if you're gonna revert this, maybe consider making the `REALLY` into `<b>really</b>` for fancy styling.

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm (hunk 25 above)
Comment (Wizardcrying) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: @9600bauds I see what these were for now. Thanks.

**Comment by Wizardcrying** [comment]
@Shadowmech88 Bauds' recent comment explained why these descriptions exist. Thanks @9600bauds.

**Comment by 9600bauds** [comment]
Again sorry for the original dumb "whyy" comment, I woke up annoyed at things entirely unrelated to spess. Just by making a proper comment we are already all communicating better.

File: code/defines/obj/weapon.dm [hunk 37]
```diff
@@ -860,7 +860,7 @@
 
 /obj/item/weapon/ectoplasm
 	name = "ectoplasm"
-	desc = "The remnants of a being between the world of the dead and that of the living. Spooky."
```
Comment (Shadowmech88) [inline code/defines/obj/weapon.dm]: Why remove this?

Comment (Wizardcrying) [reply code/defines/obj/weapon.dm]: That was a change I made. It was previously just `desc = "spooky"`. I suppose this could be a useful description.

Comment (Shadowmech88) [reply code/defines/obj/weapon.dm]: If it didn't have it originally, then I don't mind. Whatever you feel like.

Comment (Wizardcrying) [reply code/defines/obj/weapon.dm]: It is too wordy, for one.

**Comment by Wizardcrying** [comment]
That's it for this PR.

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 1 above)
Comment (Wizardcrying) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm:225]: Fixed.

**Comment by 9600bauds** [comment]
Everyone satisfied now?

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 38]
```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A plain black space pod without any distinctive markings."
 		if("pod_mil")
-			desc = "A dark grey space pod brandishing the Nanotrasen Military insignia"
```
Comment (Shadowmech88) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm:221]: "Military" may have been correct here, in the same vein as "United States Armed Forces."

Comment (Wizardcrying) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm:221]: The capitalization seemed off, since there's been no mention of the "Nanotrasen Military" from what I've seen.

**Comment by Wizardcrying** [comment]
@9600bauds The only downvoters who have commented on this are Celt and Shadowmech, so probably unless any others say something.

**Comment by ScrambledSteak** [comment]
We must delete fun at once.

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 38 above)
Comment (Intigracy) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm:221]: If it's THE Nanotrasen Military insignia, I think that's the name of it.

Comment (Intigracy) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm:221]: Or it's just laziness and not describing the insignia correctly.

**Comment by Wizardcrying** [comment]
@Aurantiaco Why do you propose this and how do you propose going about it?

**Comment by ScrambledSteak** [comment]
I say that we remove anything that causes enjoyment among players with swiftness. Fun is a toxic, horrible thing that has no reason to exist in space. My family was murdered by fun. Literally. They were playing hopscotch in the street, talking about how fun it was, when a drunk driver hit them with a car. Fun killed them.

Also this is just hyperbole I have few problems with the actual PR.

**Comment by Wizardcrying** [comment]
@Aurantiaco That's pretty sad. I'll get to work on removing fun ASAP. 

What are your specific issues?

File: code/WorkInProgress/pomf/spacepods/spacepods.dm (hunk 38 above)
Comment (Wizardcrying) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm:221]: I've never heard "Nanotrasen Military" used before, so I think it's the latter. "Nanotrasen Enforcement and Protection Division" or something shorter, perhaps?

**Comment by ScrambledSteak** [comment]
People have too much fun with

combat

being an antag

playing space

we must delete the game at once.

**Comment by Wizardcrying** [comment]
@Aurantiaco what about the PR?

**Comment by ScrambledSteak** [comment]
nah this is fine merge it

**Comment by Wizardcrying** [comment]
ok

**Review by PJB3005 — APPROVED** [review]

**Comment by ghost** [comment]
Still mispelling armour deliberately

**Comment by ultraplen** [comment]
>deleting fun flavour texts in favour of boring ones

delete this

edit: never mind, after actually reading through all the changes in the latest commit they seem fine

**Comment by 9600bauds** [comment]
Shame about part 1 not getting the FUN intervention
